src/transaction.js

```diff
@@ -386,3 +415,93 @@ export class Transaction {
     return this.instructions[0].userdata;
   }
 }
+
+/**
+ * Parse a wire transaction into a Transaction object.
+ */
+export function parseWireFormat(bytes: Buffer): Transaction {
```

Comment:
I'd prefer this to be a static method of the Transaction class, 
```js
class Transaction {
.
.
/**
 * Parse a wire transaction into a Transaction object.
 */
static from(buffer: Buffer): Transaction { ... }
.
.
}
```

Reply:
Oh duh, got it.